packages/react-aria-components/src/Calendar.tsx

```diff
@@ -448,7 +453,7 @@ export {CalendarHeaderCellForwardRef as CalendarHeaderCell};
 
 export interface CalendarGridBodyProps extends StyleProps {
   /** A function to render a `<CalendarCell>` for a given date. */
-  children: (date: CalendarDate) => ReactElement
+  children: (date: CalendarDate, weekIndex: number, dayIndex: number) => ReactElement
```

Comment:
needed to be able to determine what column a day was in for styling purposes so we can have a dashed border around ranges
better way to pass it along?

Reply:
hmm couldn't you calculate this from the date itself?

Reply:
I could, but the information already exists, so it seemed to make more sense not to do that calculation
I'll see how bad it is to calculate though

Reply:
Check commit https://github.com/adobe/react-spectrum/pull/8428/commits/aa7701d1e7011fa431a5f03c9f5bc3b9f6572ee5#diff-08206439985c4bfe014f71e8aa5df9fbe4cbcf4c478f4923e9f551cb2b4b2af7R473 to see if you like this better, or if you think of a better way to account for all the edge cases

Reply:
hmm yeah it is a bit complicated. maybe it is useful info to add to RAC. we should just consider the API. Probably should have passed an object to that function to make it extensible but would be a breaking change. I guess we could add a second options argument at least.